client/application.js

```diff
@@ -952,7 +952,7 @@ export default {
               },
               {
                 name: 'code',
-                label: 'Select the anaesthetic code you intend to use.',
+                label: 'Select the anaesthetic codes that apply to this step?',
```

Comment:
The question mark doesn't look right here. It's not a question?

Reply:
Replace it with a .